test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,37 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void anyElementsAreNull() throws Exception {
+
+        // empty list
+        assertNotNull();
+
+        // all null elements
+        assertAreNull((Object) null);
+        assertAreNull(null, null, null);
+
+        //  no null elements
+        assertNotNull(1);
+        assertNotNull(1, Integer.valueOf(1));
+
+        // mix of null and no null elements
+        assertAreNull(null, 1, Integer.valueOf(1));
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
-}
+
+    private void assertAreNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
Recall that in class, we talked about why these helper methods `assertAreNull` and `assertNotNull` are unnecessary here. The pre-existing code extracted into helper methods to avoid repeating `Arrays.asList` everywhere, but this is not the case for `anyElementsAreNull`. Do you know what we can do instead? :)